src/app/landing/landing.component.css

```diff
@@ -1,3 +1,89 @@
-.page-container{
+.landing-page.page-container{
```

Comment:
I didn't review styles but seems like you should use ["View encapsulation"](https://angular.io/guide/view-encapsulation) to avoid duplicates of `common.css` imports.